src/libfsm/determinise.c

```diff
@@ -618,3 +749,18 @@ fsm_determinise(struct fsm *fsm)
 	return r;
 }
 
+int
+fsm_glushkovize(struct fsm *fsm)
+{
+	struct fsm_determinise_cache *dcache;
+	int r;
+
+	dcache = NULL;
+
+	r = nfa_transform_cache(fsm, &dcache, NFA_XFORM_GLUSHKOVIZE);
+
+	fsm_determinise_freecache(fsm, dcache);
+
+	return r;
+}
+
```

Comment:
Would prefer to move out the guts to its own file, and have both the `fsm_determinise*()` and `fsm_glushkovize*()` interfaces in fsm_determinise.c and fsm_glushkovize.c respectively, calling into it. That'll help make naming clearer too, I hope.

But it'll make for a messy diff, so maybe on a separate PR?

Reply:
yes, I think a separate PR would be best.